addons/expanded-backpack/backpack.css

```diff
@@ -0,0 +1,31 @@
+.backpack_backpack-list_MOp5X {
```

Comment:
Change this to `[class^="backpack_backpack-list"]`
Do this for every CSS selector

By the way, @mxmou When is it suitable to target full hashed class names? Addons such as editor-dark-mode intentionally do not, as far as I know. At some point we should write about this on the developer docs.

Reply:
The hashes don't work in TurboWarp Desktop so they should never be included in the selector. This isn't a problem for editor dark mode because it doesn't exist in TurboWarp.